docs/How-to-release.md

```diff
@@ -119,99 +130,99 @@ Guide to build the release from source :
 
  * https://github.com/apache/skywalking-cli/blob/v$VERSION/docs/How-to-release.md
 
-Voting will start now (xxxx date) and will remain open for at least 72 hours, all PMC members are required to give their votes.
+Voting will start now and will remain open for at least 72 hours, all PMC members are required to give their votes.
+
 [ ] +1 Release this package.
 [ ] +0 No opinion.
 [ ] -1 Do not release this package because....
+
+Please take part in the vote process.
+
+Everyone could vote (though may be no-binding), but make sure to check the list [1].
```

Comment:
`check the list` is only recommended not required. Even PMC members could just choose a subset to check some of them.  
The basic line is making sure the release following ASF requirements and doesn't bring any IP and LICENSE issue.

Reply:
ok, deleted